test/e2e/compile.e2e.js

```diff
@@ -550,6 +551,120 @@ describe('Compile Commands', () => {
 		expect(exitCode).to.equal(1);
 	});
 
+	it ('Creates a bundle with legacy flat project', async () => {
+		const platform = 'photon';
+		const cwd = path.join(PATH_FIXTURES_THIRDPARTY_OTA_DIR, 'projects', 'stroby-with-assets');
+		const destinationZip = path.join(PATH_TMP_DIR, 'bundle.zip');
+		const args = ['compile', platform, '--saveTo', destinationZip];
+		const { stdout, stderr, exitCode, start, end } = await cliRunWithTimer(args, { cwd });
+		const file = await fs.stat(destinationZip);
+		const log = [
+			`Compiling code for ${platform}`,
+			'',
+			'Including:',
+			'    src/stroby.ino',
+			'    project.properties',
+			'',
+			'Including:',
```

Comment:
This will need to be updated based on my comment combining the source and assets list.